docs/data/charts/tree-map/tree-map.md

```diff
@@ -7,9 +7,10 @@ title: Charts - Tree map
 
 <p class="description">Tree map allows to display data with a hierarchical structure.</p>
 
-> ⚠️ This feature isn't implemented yet. It's coming.
->
-> 👍 Upvote [issue #7924(https://github.com/mui/mui-x/issues/7924 if you want to see it land faster.
->
-> 💬 To have a solution that meets your needs, leave a comment on the [same issue](https://github.com/mui/mui-x/issues/7924.
-> If you already have a use case for this component, or if you are facing a pain-point with your current solution.
+:::warning
+This feature isn't implemented yet. It's coming.
+
+👍 Upvote [issue #7924](https://github.com/mui/mui-x/issues/7924) if you want to see it land faster.
+
+Don't hesitate to leave a comment on the same issue to influence what gets built. Especially if you already have a use case for this component, or if you are facing a pain point with your current solution.
+:::
```

Comment:
@alexfauquette @samuelsycamore I think that we could migrate all the uses of `<blockquotes>` in our docs when it comes to callouts. In the spirit of HTML, `>` is meant to be a quote: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/blockquote

Reply:
I agree `:::warning` is more relevant than `>`. For me except in blog article I don't see usage of `>`

Reply:
> I think that we could migrate all the uses of <blockquotes> in our docs when it comes to callouts.

I could've sworn I actually did this late last year but I must have lost track of the PR in the X repo at some point. 😅